projects/onnx/build.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+# Copyright 2023 ISP RAS
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+set -e
+
+cp -r /onnx /onnx_${TARGET}
+cd /onnx_${TARGET}
+
+# Clean previous build
+python3 setup.py clean
+
+echo "Build directory clean"
+
+CMAKE_ARGS="-DONNX_USE_PROTOBUF_SHARED_LIBS=OFF -DONNX_ML=ON -DONNX_NAMESPACE=onnx"
+
+# Build Onnx.
+if [[ $TARGET == "fuzz" ]]
+then
+    CC=clang
+    CXX=clang++
+    CFLAGS="-fPIC -g -fsanitize=fuzzer-no-link,address,bounds,integer,undefined,null,float-divide-by-zero"
+    CXXFLAGS=$CFLAGS
+elif [[ $TARGET == "afl" ]]
+then
+    CC=afl-clang-fast
+    CXX=afl-clang-fast++
+    # '-fno-sanitize=pointer-overflow' is a fix for clang-14 (https://github.com/llvm/llvm-project/issues/60442).
+    CFLAGS="-fPIC -g -fsanitize=address,bounds,integer,undefined,null,float-divide-by-zero -fno-sanitize=pointer-overflow"
+    CXXFLAGS=$CFLAGS
+elif [[ $TARGET == "sydr" ]]
+then
+    CC=clang
+    CXX=clang++
+    CFLAGS="-fPIC -g"
+    CXXFLAGS=$CFLAGS
+elif [[ $TARGET == "cov" ]]
+then
+    CC=clang
+    CXX=clang++
+    CFLAGS="-fPIC -g -fprofile-instr-generate -fcoverage-mapping"
+    CXXFLAGS=$CFLAGS
+fi
+
+CMAKE_ARGS=$CMAKE_ARGS CC=$CC CXX=$CXX CFLAGS=$CFLAGS CXXFLAGS=$CXXFLAGS MAX_JOBS=$(nproc) \
+    python3 setup.py build
+CMAKE_ARGS=$CMAKE_ARGS CC=$CC CXX=$CXX CFLAGS=$CFLAGS CXXFLAGS=$CXXFLAGS MAX_JOBS=$(nproc) \
+    python3 setup.py install_headers
+
+# Build targets.
+TARGET_FLAGS=""
+MAIN=""
+if [[ $TARGET == "fuzz" || $TARGET == "afl" ]]
+then
+    TARGET_FLAGS="-fsanitize=fuzzer,address,bounds,integer,undefined,null,float-divide-by-zero"
+else
+    $CXX /main.cc -c -o ./main.o
+    MAIN="./main.o"
+fi
+
+if [[ $TARGET == "cov" ]]
+then
+    TARGET_FLAGS="-fprofile-instr-generate -fcoverage-mapping"
+fi
+
+build_target(){
+    target=$1
+    extra=$2
```

Comment:
second argument `extra` is redundant here, because it is constant for all fuzztargets. These flags can be specified directly in the build command.